README.md

```diff
@@ -98,40 +121,71 @@ After configuration, build with `cmake --build <build_dir> -- -j<nproc>`
 ### Logger tests
 
 Tests API implementation of logger verbosity and functionality.
-o <build_dir>/bin/logger_test
+
+```bash
+  <build_dir>/bin/logger_test
+```
 
 ## Running Contraction Tests
 
 ### Bilinear contraction tests
```

Comment:
Can we make this heading (and Scale contraction tests) into bullet points in order to avoid stacked headings?